docs/docsite/rst/playbooks_loops.rst

```diff
@@ -277,10 +277,12 @@ The authorized_key pattern is exactly where it comes up most.
 Looping over Integer Sequences
 ``````````````````````````````
 
-``with_sequence`` generates a sequence of items in ascending numerical order. You
-can specify a start, end, and an optional step value.
+``with_sequence`` generates a sequence of items. You
+can specify a start value, an end value, an optional "stride" value that specifies the number of steps to increment the sequence, and an optional printf-style format string.
 
-Arguments should be specified in key=value pairs.  If supplied, the 'format' is a printf style string.
+Arguments should be specified as key=value pair strings.
```

Comment:
@bcoca actually, no, you can't use YAML format. I double checked it.

in case of:
```
      with_sequence:
        - count: 4
```
the error is
> unknown error parsing with_sequence arguments: {u'count': 4}. Error was: expected string or buffer

in case of:
```
      with_sequence:
        count: 4
```
the error is
> unrecognized arguments to with_sequence: [u'_raw_params']

I tried to add easy fix to achieve list-dict syntax like in the docs by doing:
```
                    if isinstance(term, dict):
                        self.parse_kv_args(term) # don't parse k=v if term is a dict
                    else:
                        if not self.parse_simple_args(term):
                            self.parse_kv_args(parse_kv(term))
```
but this still requires to rewrite `parse_kv_args` because of
> Error was: int() can't convert non-string with explicit base

There's open issue #13879 about this.